pkgs/development/tools/misc/gdb/wrapper.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, makeWrapper, gdb-unwrapped, safePaths }:
+
+let
+  gdb = gdb-unwrapped;
+in
+  stdenv.mkDerivation {
+    name = gdb.name;
+    buildInputs = [ makeWrapper ];
+    propagatedBuildInputs = [ gdb ];
```

Comment:
Why propagate it? Do packages need the actual `gdb` when they have `gdb` as `buildInputs`? Packages typically don't have `gdb` in `buildInputs` either.